agent/lib/kontena/workers/log_worker.rb

```diff
@@ -20,14 +20,15 @@ class LogWorker
     QUEUE_MAX_SIZE = 2000
     QUEUE_THROTTLE = (QUEUE_MAX_SIZE * 0.8)
     WATCH_INTERVAL = 10.0
+    BATCH_SIZE = 10
```

Comment:
Would a higher batch size work better? Is there some reason to limit the batches to such a small number?

Reply:
I experimented with different sizes, higher batch size does not make things significantly better .. and there is small risk that server reactor pauses for longer period with bigger batch.